.github/workflows/l2_prover_ci.yaml

```diff
@@ -2,12 +2,8 @@ name: L2 Prover CI
 on:
   push:
     branches: ["main"]
-    paths:
-      - "crates/l2/prover/**"
```

Comment:
We may trigger the CI even if it's not strictly necessary, which could make the merging process take longer. Maybe we can set it to "crates/l2/**" ?

Reply:
Agreed, let's restore this and then we can merge

Reply:
The bug was introduced because the breaking change was made in another directory, outside of `crates/`, I think we should be running every test each time to avoid predicting where a breaking change could occur.

Reply:
That's fair, the workflow also does not take too long so I think it's fine to leave it